tools/pages/permit-holders/filter-permit-holders.ts

```diff
@@ -0,0 +1,38 @@
+import { gql } from '@apollo/client'; // gql tag
+import { ApplicantsFilter, Applicant } from '@lib/graphql/types';
+
+export const FILTER_PERMIT_HOLDERS_QUERY = gql`
```

Comment:
```suggestion
export const GET_PERMIT_HOLDERS_QUERY = gql`
```
Still a get request at the end of the day